### PR TITLE
Transpose of Matrix in Java 

### PR DESCRIPTION
**Pull request by mrunalthosar57**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description
Problem Statement: Transpose of matrics
Description:
Transpose of the matrix is nothing but swapping of the rows and columns in the given matrix.
Solution:
n: number of the rows in the matrix
m: number of the columns in the matrix
n x m: the size of the matrix

Input:
n: 2
m: 2
Input matrix:
1 2
3 4

Output:
Transpose of the matrix:
1 3
2 4

### Checklist

- [Yes] I've read the contribution guidelines.
- [Yes] I've checked the issue list before deciding what to submit.
- [Yes] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#4170

### CODE REVIEW

**Comment by mrunalthosar57** [comment]
Hello @HarshCasper, @bhav09, @gat786, @Apoorve73,
Kindly review my PR.

File: Java/math/Matrix_Transpose.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
+/*
+* Matrix is the Two-Dimensional Model
+* Matrix is the combination of rows and columns,here the rows are n & columns are m
+* size = n x m
+* Transpose of the matrix is swapping of the rows and columns in the given matrix
+* Following code gives the output of the transpose of the matrix
+*/
+
+package First;
+
```
Comment (ankitaggarwal23) [inline Java/math/Matrix_Transpose.java]: Remove the package

File: Java/math/Matrix_Transpose.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+import java.util.Scanner;
+
+public class Matrix_Transpose 
+{
+	public static void transpose(int[][] a,int[][] b, int n , int m) //Function for the transpose
```
Comment (ankitaggarwal23) [inline Java/math/Matrix_Transpose.java]: Shift the comment above the function

File: Java/math/Matrix_Transpose.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+		System.out.println("Enter number of columns:");
+		int m = sc.nextInt(); 
+		
+		int i=0,j=0;
+		
+		//Assigning the memory to the 2-D array according to given matrix size
+		int[][] a = new int[n][m];
+		int[][] b = new int[m][n]; 
+       
+
```
Comment (ankitaggarwal23) [inline Java/math/Matrix_Transpose.java]: Remove the one extra line

File: Java/math/Matrix_Transpose.java [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+		{
+			for(j=0;j<m;j++)
+			{
+				//To display elements of the matrix that are taken from user
+				System.out.print(a[i][j]+" ");
+			}
+
+			System.out.println();
+		}
+	   	transpose(a, b , n , m); // Function call to calculate the transpose
```
Comment (ankitaggarwal23) [inline Java/math/Matrix_Transpose.java]: Remove the redundant comment

File: Java/math/Matrix_Transpose.java [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+Explanation:
+*Inputs, rows are 2 and columns are 3 
+* On the next line all the elements of the matrix have been entered
+10 20 30 40 50 60
+* Then, the code gives output for the transpose.
+ 
+ 
+ Time Complexity : theta(nm) where, n = number of rows & m = number of columns
+ Space Complexity : Transpose matrix requires extra memory space, theta(nm)
+*/
```
Comment (ankitaggarwal23) [inline Java/math/Matrix_Transpose.java:115]: Add a new line at the EOF

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by mrunalthosar57** [comment]
Hello @ankitaggarwal23 ,
changes are made,kindly review it.

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**